modules/user_data/templates/01_disk_management.sh.tpl

```diff
@@ -35,7 +35,7 @@ for i in $(seq 1 6); do
   )
 
   if [ -z "$${VOLUME_ID:-}" ]; then
-    echo 'EBS volume not yet available'
+    echo 'ebs volume not yet available'
```

Comment:
Why ? EBS is an abbreviation

Reply:
maybe something happened during the rebase and it picked some of the old changes.